src/HPWH.in.hh

```diff
@@ -1000,6 +1002,11 @@ private:
 	/// Generates a vector of logical nodes
 	std::vector<HPWH::NodeWeight> getNodeWeightRange(double bottomFraction,double topFraction);
 
+	/// True: water is drawn from the tank itself; False: tank provides heat exchange only
+	bool waterIsDrawnFromTank;
```

Comment:
Consider renaming. Thoughts:

- `hasHeatExchanger`
- make enum called `drawScheme` with options of `fromTank` and `throughHeatExchanger`
- Other ideas?

Reply:
We could subclass heat-exchange models and have a standard function for heating the tank and an override version  for heat exchange. This would eliminate a couple of unused parameters in the HPWH class.

Reply:
I think we ultimately want to move in this direction, but it should be part of a larger refactor where we are separating the components into their own classes. I could see having a separate heat exchanger class altogether.